operator/src/main/java/oracle/kubernetes/operator/logging/LoggingFormatter.java

```diff
@@ -134,14 +132,13 @@ public String format(LogRecord record) {
 
     } catch (JsonProcessingException e) {
       String tmp =
-          "{\"@timestamp\":%1$s,\"level\":%2$s, \"class\":%3$s, \"method\":\"format\", \"timeInMillis\":%4$d, "
+          "{\"@timestamp\":%1$s,\"level\":%2$s, \"class\":%3$s, \"method\":\"format\", "
               + "\"@message\":\"Exception while preparing json object\",\"exception\":%5$s}\n";
       return String.format(
           tmp,
           dateString,
           level,
           LoggingFormatter.class.getName(),
-          rawTime,
```

Comment:
why remove 'rawTime'? (it doesn't seem to hurt to include it, and it's sometimes useful)

Reply:
This isn't for ordinary log messages, but is the catch block for what I suspect is the rare occurrence where we have a JSON exception during the generation of the normal messages. Is it helpful in that use case? Unless I missed it, this raw time doesn't seem to be part of the normal log message.

Reply:
`    map.put(TIME_IN_MILLIS, rawTime);` is removed in this pull, which presumably has a side effect on the code in the following try
```
     ObjectMapper mapper = new ObjectMapper();
      json = mapper.writeValueAsString(map);
...
    return json + "\n";

```

Reply:
In other words, it looks to me like it shows up in the 'normal' path too when the `map.put` is in place...

Reply:
Okay, I'll re-add it... I apparently need more ☕ for my reading comprehension these days.